demo/index.html

```diff
@@ -273,11 +273,22 @@ <h3>explicitly sized <code>cosmoz-tabs</code> with overflowing tab content shows
 		<script type="text/javascript">
 		(function () {
 			window.addEventListener('WebComponentsReady', function () {
-				var basic = document.getElementById('basic');
-				basic = !basic.set ? basic.firstElementChild : basic;
-				basic.stringify = function (values) {
-					return JSON.stringify(values);
-				};
+				(function () {
+					var basic = document.getElementById('basic');
+					basic = !basic.set ? basic.firstElementChild : basic;
+					basic.stringify = function (values) {
+						return JSON.stringify(values);
+					};
+				})();
+
+				(function () {
+					var hashParam = document.getElementById('hashParam');
+					hashParam = !hashParam.set ? hashParam.firstElementChild : hashParam;
```

Comment:
This does not work because the `hashParam` id is set on the template instead of the `dom-bind` element.